terraform/environments/mlra/cloudwatch.tf

```diff
@@ -10,21 +9,13 @@ data "aws_secretsmanager_secret_version" "pagerduty_integration_keys" {
 }
 
 module "cwalarm" {
-  source = "./modules/cloudwatch"
-
-  pClusterName          = local.application_name
-  pAutoscalingGroupName = "${local.application_name}-cluster-scaling-group"
-  pLoadBalancerName     = module.alb.load_balancer.arn_suffix
-  pTargetGroupName      = module.alb.target_group_name
-  appnameenv            = "${local.application_name}-${local.environment}"
-  snsTopicName          = local.sns_topic_name
-}
-
-module "pagerduty_core_alerts" {
-  depends_on = [
-    module.cwalarm
-  ]
-  source                    = "github.com/ministryofjustice/modernisation-platform-terraform-pagerduty-integration?ref=v1.0.0"
-  sns_topics                = [local.sns_topic_name]
+  source                    = "./modules/cloudwatch"
+  tags                      = local.tags
+  pClusterName              = local.application_name
+  pAutoscalingGroupName     = "${local.application_name}-cluster-scaling-group"
+  pLoadBalancerName         = module.alb.load_balancer.arn_suffix
+  pTargetGroupName          = module.alb.target_group_name
+  appnameenv                = "${local.application_name}-${local.environment}"
+  sns_topic_name            = "${local.application_name}-${local.environment}-alerting-topic"
   pagerduty_integration_key = local.pagerduty_integration_keys[local.pagerduty_integration_key_name]
```

Comment:
MP-provided module modernisation-platform-terraform-pagerduty-integration is no longer use, since it only contains one resource, the `aws_sns_topic_subscription` which is not defined in our local cloudwatch module